akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Polls.scala

```diff
@@ -561,12 +561,12 @@ object PollFactory {
     questionOption
   }
 
-  def createPoll(id: String, pollType: String, numRespondents: Int, answers: Option[Seq[String]]): Option[Poll] = {
+  def createPoll(id: String, pollType: String, numRespondents: Int, answers: Option[Seq[String]], title: Option[String]): Option[Poll] = {
```

Comment:
Does `title` makes reference to the same entity as `question`?

Reply:
It is the same as the question of pollStarted and pollPublished events, but in the Scala classes there is already a Question structure (a poll could have multiple questions), and there was already this `title` option for the Poll, so I used it.
Yes it is confusing, I think `question` name in pollStarted/Published  should be renamed to title also.

Reply:
Yeah, whatever you feel makes more sense. I agree we should keep a unique name.

Reply:
I think a way ahead would be to create a github 'refactor' issue for renaming 'question' to 'title' and then once created, to mention it here and merge this PR.